docs/rest_api/pages.rst

```diff
@@ -0,0 +1,462 @@
+
+Get an individual Landing Page by ID.
+
+.. vale off
+
+**HTTP Request**
+
+.. vale on
+
+``GET /pages/ID``
+
+**Response**
+
+``Expected Response Code: 200``
+
+.. code-block:: json
+
+   {
+       "page": {
+           "id": 3,
+           "title": "Webinar Landing Page",
+           "description": null,
+           "isPublished": true,
+           "publishUp": null,
+           "publishDown": null,
+           "dateAdded": "2015-07-15T15:06:02-05:00",
+           "createdBy": 1,
+           "createdByUser": "Joe Smith",
+           "dateModified": "2015-07-20T13:11:56-05:00",
+           "modifiedBy": 1,
+           "modifiedByUser": "Joe Smith",
+           "category": "Events",
+           "language": "en",
+           "template": "blank",
+           "customHtml": "<!DOCTYPE ...",
+           "hits": 0,
+           "uniqueHits": 0,
+           "variantHits": 0,
+           "revision": 1,
+           "metaDescription": null,
+           "redirectType": null,
+           "redirectUrl": null,
+           "translationChildren": [],
+           "translationParent": null,
+           "variantChildren": [],
+           "variantParent": null,
+           "variantSettings": [],
+           "variantStartDate": null
+       }
+   }
+
+**Landing Page Properties**
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Description
+   * - ``id``
+     - int
+     - ID of the Landing Page
+   * - ``title``
+     - string
+     - Title of the Landing Page
+   * - ``description``
+     - string/null
+     - Description of the Landing Page
+   * - ``alias``
+     - string
+     - Used to generate the URL for the Landing Page
+   * - ``isPublished``
+     - boolean
+     - Published state
+   * - ``publishUp``
+     - datetime/null
+     - Landing Page publish date/time
+   * - ``publishDown``
+     - datetime/null
+     - Landing Page unpublish date/time
+   * - ``dateAdded``
+     - ``datetime``
+     - Landing Page creation date/time
+   * - ``createdBy``
+     - int
+     - ID of the User that created the Landing Page
+   * - ``createdByUser``
+     - string
+     - Name of the User that created the Landing Page
+   * - ``dateModified``
+     - datetime/null
+     - Date/time Landing Page was last modified
+   * - ``modifiedBy``
+     - int
+     - ID of the User that last modified the Landing Page
+   * - ``modifiedByUser``
+     - string
+     - Name of the User that last modified the Landing Page
+   * - ``language``
+     - string
+     - Language locale of the Landing Page
+   * - ``template``
+     - string
+     - Template of the Landing Page
+   * - ``customHtml``
+     - string
+     - Static HTML of the Landing Page
+   * - ``hits``
+     - int
+     - Total Landing Page hit count
+   * - ``uniqueHits``
+     - int
+     - Unique Landing Page hit count
+   * - ``revision``
+     - int
+     - Landing Page revision
+   * - ``metaDescription``
+     - string
+     - Meta description for the Landing Page's ``<head>``
```

Comment:
Does this need to be plural? If so should the others?

Reply:
Nope, singular in this case. This API endpoint allows for retrieving a single Landing Page 😊 To improve clarity a bit, I'll change this to:

> Meta description which is put in the Landing Page's ``<head>`` element.